wofs/wofs_app.py

```diff
@@ -456,30 +464,47 @@ def estimate_job_size(num_tasks):
 @click.group(help='Datacube WOfS')
 @click.version_option(version=__version__)
 def cli():
+    """
+    Instantiate a click.group object
```

Comment:
When commenting code, it's important that they're a useful addition and not just repeating what the code already says. In this case, the code is self explanatory, and the docstring could explain *why* we have a click group (to be able to provide a set of sub-commands for different bits of WOfS processing). Or even User documentation for how to use the WOfS command...